src/Main.py

```diff
@@ -0,0 +1,678 @@
+            for child in self.carousel_questions.get_children():
+                self.carousel_questions.remove(child)
+            if self.get_rootfs(widget) == None:
+                return
+            if self.get_user(widget) == None:
+                return
+            self.btn_show_log.clicked()
+            self.post_command = post
+            if self.rootfs.root_subvol == None:
+                self.vte_command("env disk={} pardus-chroot /dev/{} su {} -".format(self.rootfs.name, self.rootfs.name, self.user))
+            else:
+                self.vte_command("env subvolume={} pardus-chroot /dev/{} su {} -".format(self.rootfs.root_subvol ,self.rootfs.name, self.user))
+            self.user = None
+            self.pending_func = None
+        def post():
+            self.btn_close_logs.clicked()
+            self.update_status_page(_("Chroot Process Successfully Concluded"), "emblem-ok-symbolic", _("The chroot process has finished successfully"), True, True)
+        pre()
+
+    def update_status_page(self, title, icon_name, description, stop_spinner=False, enable_mainpage=True):
+        self.status_page.set_title(title)
+        self.status_page.set_icon_name(icon_name)
+        self.status_page.set_description(description)
+        self.btn_go_mainpage.set_sensitive(enable_mainpage)
+        self.spinner_loading.start()
+        if stop_spinner:
+            self.spinner_loading.stop()
+
+    def vte_command(self, command):
+        try:
+            env_vars = [f'{key}={value}' for key, value in os.environ.items()]
+            self.vte_terminal.reset(True, True)
+            exec = self.vte_terminal.spawn_async(
+                Vte.PtyFlags.DEFAULT, os.environ['HOME'], ["/bin/bash", "-c", command], env_vars, GLib.SpawnFlags.SEARCH_PATH, None, None, -1, None)
+        except Exception as e:
+            # write error to stderr
+            sys.stderr.write(str(e) + "\n")
+            self.update_status_page(_("An error occured"), "dialog-error-symbolic", str(e), True, True)
+
+    def vte_cb(self, pid, error):
+        if error != None or pid == -1:
+            self.update_status_page(_("An error occured"), "dialog-error-symbolic", _("An error occurred before the command has been executed"), True, True)
+            return
+    def vte_exited(self, widget, status):
+        exit_status = os.waitstatus_to_exitcode(status)
+        if exit_status != 0:
+            self.update_status_page(_("An error occured"), "dialog-error-symbolic", _("An error occured while executing the command. Please check the logs"), True, True)
+            return
+        if self.post_command == None:
+            self.update_status_page(_("An error occured"), "dialog-error-symbolic", _("An error occurred after the command has been executed."), True, True)
+            return
+        Thread(target=self.post_command).start()
```

Comment:
You can change automatically status page if log page shown.

Reply:
done: 8c14a96